crates/duty-tracker/src/contract_manager.rs

```diff
@@ -40,8 +51,7 @@ impl ContractManager {
                     })
                     .collect::<BTreeMap<Txid, ContractSM>>(),
                 Err(_) => {
-                    debug_assert!(false, "Failed to load contracts");
-                    BTreeMap::new()
+                    todo!() // TODO(proofofkeags): probably wanna crash here?
```

Comment:
Fine to crash for now.
But should try to back off for a `const` configurable time and then try again for `const` configurable number of times; and then CRASH!